source/features/clean-repo-sidebar.tsx

```diff
@@ -0,0 +1,32 @@
+import './clean-repo-sidebar.css';
+
+import select from 'select-dom';
+import elementReady from 'element-ready';
+import * as pageDetect from 'github-url-detection';
+
+import features from '.';
+
+async function init(): Promise<void> {
+	// Hide empty meta if it’s not editable by the current user
+	// TODO: once it's fixed, use https://github.com/fregante/github-url-detection/blob/4840d85d31d59bfe71e884ef42c482fbfff2d955/index.ts#L490
+	if (!select.exists('.repository-content [aria-label="Edit repository metadata"]')) {
+		select('.repository-content .BorderGrid-cell:first-child > .text-italic')?.remove();
```

Comment:
This only hides the description, but leaves an empty "About" header. That should be removed too, like it happens in `clean-conversation-sidebar`. This needs to happen cleanly, so the whole section disappears and the content below appears in its place.

Reply:
> This only hides the description, but leaves an empty "About" header.

The "About" section isn't totally empty, there's still the repo age in it.

> Also is the `?.` necessary or can it be `!.`?

I think it is, given that the text element only exists on repos with no meta infos.

Reply:
> The "About" section isn't totally empty, there's still the repo age in it

That's true. I think it wasn't showing for me because I had not set the token